web-app/main.js

```
@@ -0,0 +1,225 @@
+clearChatBtn.addEventListener('click', () => {
+  chatMessages.innerHTML = '';
+  quickReplies.innerHTML = '';
+  wizard.active = false;
+  wizard.step = 0;
+  wizard.answers = {};
+  addMessage('Conversa limpa. Quando quiser, clique em "Novo Projeto Guiado".', 'agent');
+});
+
+tabs.forEach((tab) => {
+  tab.addEventListener('click', () => {
+    tabs.forEach((item) => item.classList.remove('active'));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)